public/locales/tr/send.ftl

```diff
@@ -15,6 +16,7 @@ encryptingFile = Şifreleniyor...
 decryptingFile = Şifre çözülüyor...
```

Comment:
Thanks for looking into this. Let me send an email to the mailing list and ask locales to look into it, it's a also a way to measure how much they follow communication (most of these locales should).